system-tests/_support/utils/StringUtil.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+
+  /**
+   * Return the element's text input, remove duplicate whitespaces, replace
+   * newlines with spaces and convert it to lower-case.
+   *
+   * @param {DOMElement} element - The HTML DOM element to extract the text from
+   * @returns {String} Returns the normalized string contents
+   */
+  getNormalizedContents(element) {
```

Comment:
Replaced with `.getContents` and I like the idea for using it in conjunction with `.toLowerCase` 👍